src/kubetop/_metadata.py

```diff
@@ -1,3 +1,3 @@
 
 version_tuple = (17, 4, 17, 1)
-version_string = ".".join(map(str, version_tuple) + ["dev0"])
+version_string = ".".join([str(v) for v in version_tuple] + ["dev0"])
```

Comment:
Can you use `list(...)` instead of `[...]` so that the loop variable doesn't leak on Python 2?  Thanks.